content/en/tracing/trace_pipeline/ingestion_mechanisms.md

```diff
@@ -196,13 +196,23 @@ Read more about sampling controls in the [Node.js tracing library documentation]
 [1]: /tracing/trace_collection/dd_libraries/nodejs
 {{% /tab %}}
 {{% tab "PHP" %}}
+**Remote configuration**
+
+<div class="alert alert-info"><strong>Resource-based sampling rules are in Beta</strong>: Starting from version <a href="https://github.com/DataDog/dd-trace-php/releases/tag/1.4.0">1.4.0</a>, for PHP applications, set by-service and by-resource sampling rates from the <a href="/tracing/trace_pipeline/ingestion_controls#configure-the-service-ingestion-rate">Ingestion Control Page</a> UI. Request access to the feature via this <a href="https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/">link</a>.</div>
```

Comment:
```suggestion
<div class="alert alert-info"><strong>Resource-based sampling rules are in Preview</strong>. Starting from version <a href="https://github.com/DataDog/dd-trace-php/releases/tag/1.4.0">1.4.0</a>, for PHP applications, set by-service and by-resource sampling rates from the <a href="https://app.datadoghq.com/apm/traces/ingestion-control">Ingestion Control Page</a>. Use <a href="https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/">this form</a> to request access today.</div>
```

Reply:
I copy pasted that part from the Java tab some lines higher. I'd suggest you improve this there as well then. I think the texts should be consistent at least.

Reply:
@bwoebi thanks for flagging that, the other lines have already been updated in the following PR:
https://github.com/DataDog/documentation/pull/25376